### PR TITLE
Set volume dir SELinux context if possible

### LINKED ISSUE #741: SELinux problems with volumes when running as a Docker container
Running OpenShift as a docker (1.4.1) container on fc21 with:

````
docker run -d --name=openshift -v /var/run/docker.sock:/var/run/docker.sock --privileged --net=host openshift/origin:latest start
````

Even without mounting any volume, the container uses `/var/lib/openshift` on the host (not quite sure how it does that, but hey!) for managed volumes.

Since the latest RPM upgrade (`docker-io-1.4.1-5`) anything using an kubernetes volume fails to start up unless I run on the host:

````
chcon -Rt svirt_sandbox_file_t /var/lib/openshift/openshift.local.volumes/
````

Is there another way to work around this?

### PR DESCRIPTION
**Pull request by ncdc**
Fixes #741

### CODE REVIEW

**Comment by ncdc** [comment]
[test]

**Comment by ncdc** [comment]
@smarterclayton what sort of tests do you want for this?

File: pkg/cmd/server/kubernetes/node.go [hunk 1]
```diff
@@ -89,6 +90,14 @@ func (c *NodeConfig) EnsureVolumeDir() {
 		if mkdirErr := os.MkdirAll(rootDirectory, 0750); mkdirErr != nil {
 			glog.Fatalf("Couldn't create kubelet volume root directory '%s': %s", rootDirectory, mkdirErr)
 		}
+		if chconPath, chconLookupErr := exec.LookPath("chcon"); chconLookupErr != nil {
+			glog.V(2).Infof("Couldn't locate 'chcon' to set the kubelet volume root directory context: %s", chconLookupErr)
+		} else {
+			chconCmd := exec.Command(chconPath, "-t", "svirt_sandbox_file_t", rootDirectory)
+			if chconRunErr := chconCmd.Run(); chconRunErr != nil {
```
Comment (smarterclayton) [inline pkg/cmd/server/kubernetes/node.go]: Use err instead of the longer err name

Comment (ncdc) [inline pkg/cmd/server/kubernetes/node.go]: Updated

**Comment by smarterclayton** [comment]
Mock the command executor (maybe split the init function off) and verify it makes a call.  That's about all we can do.

**Comment by ncdc** [comment]
We don't have any other tests for pkg/cmd/server/kubernetes/node.go at the moment... do you want to merge this as-is, or wait for me to refactor the command executor and write some tests?

**Comment by smarterclayton** [comment]
If you can do it quickly, yes

**Comment by ncdc** [comment]
I'll see what I can do

**Comment by smarterclayton** [comment]
Thank you LGTM [merge]

**Pull request merged**